tests/vm

```diff
@@ -11,15 +11,15 @@ IMAGE="${TEST_IMG:-ubuntu-minimal-daily:24.04}"
 
 # Launch test instance
 lxc launch "${IMAGE}" vm1 --vm
-waitInstanceBooted vm1
+waitInstanceReady vm1
```

Comment:
Couldn't this break `lxc exec` entirely?

Reply:
Right, indeed I confused the 2. It was already using the longer form (`waitInstanceBooted` includes `waitInstanceReady`). I dropped that commit, thanks for flagging it!